### ISSUE TITLE
Endpoints return strings instead of objects

### ISSUE THREAD

**Issue by lenovouser** [jdalrymple/gitbeaker#1418]
**Steps to reproduce**

```ts
import * as gitlab from '@gitbeaker/node';

const api = new gitlab.Gitlab({
    host: 'https://gitlab.example.com',
    token: '123456789123456789123456789',
});

void (async () => {
    const group = await api.Groups.show(1);

    console.log(typeof group);
})();
```

```
experiments on  master [✘!?] via ⬢ v14.15.3
➜ ts-node gitlab.ts
string
```

**Expected behaviour**

That it returns me an object with of the type `GroupDetailSchema`.

**Actual behaviour**

That it returns me a `string`

**Comment by lenovouser** [comment]
Also happens when I don't use `ts-node`:

```js
const gitlab = require('@gitbeaker/node');

const api = new gitlab.Gitlab({
    host: 'https://gitlab.example.com',
    token: '123456789123456789123456789',
});

void (async () => {
    const group = await api.Groups.show(1);

    console.log(typeof group);
})();
```

```sh
experiments on  master [✘!?] via ⬢ v14.15.3
❯ node server-creation/gitlab.js
string
```

**Comment by lenovouser** [comment]
Node: `14.15.3` (But also `15.5.0`)
Library: `"@gitbeaker/node": "27.0.0"`

**Comment by lenovouser** [comment]
Also happening with `"@gitbeaker/node": "28.0.0"`

**Comment by jdalrymple** [comment]
Cant seem to recreate this behavior, could you put together a test repo. Running this test locally works fine. Did you actually use a group with id 1? Is it possible you don't have access to the group? If thats the case, i should update the error message given.

**Comment by MaximeMorille** [comment]
@lenovouser can you check the content type of the response ? I have the same issue with my Cypress tests, since Cypress adds `chartset=utf-8` in stubbed content types.

@jdalrymple would it be possible to split the response's content type by `;` to keep only the media type and remove optional boundary/charset of the content type header, in [`processBody`](https://github.com/jdalrymple/gitbeaker/blob/master/packages/gitbeaker-browser/src/KyRequester.ts) ?

**Comment by jdalrymple** [comment]
@MaximeMorille  What is being returned that contains a ';' ?

**Comment by MaximeMorille** [comment]
I had the following code in my Cypress tests :

```javascript
    // Stub a request and set its response to custom data
    cy.route({
        method: 'GET',
        url: '**/issues**',
        response: data,
        headers: {
            'Content-Type': 'application/json',
        },
    }).as('issues');
```

With this configuration, Cypress seems to add `; charset=utf-8` to the `Content-Type` header, which is then `Content-Type: application/json; charset=utf-8`.

I fixed my tests by removing the `headers` from the route's configuration, but it may be safer to ensure that Gitbeaker uses only the media type, and not the complete Content Type to parse the response. What do you think ?

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`28.0.3`](https://github.com/jdalrymple/gitbeaker/releases/tag/28.0.3) :rocket:
